src/main/java/cz/cvut/kbss/study/util/RoleAssignmentUtil.java

```diff
@@ -0,0 +1,78 @@
+package cz.cvut.kbss.study.util;
+
+import cz.cvut.kbss.study.model.Vocabulary;
+
+import java.util.*;
+
+public class RoleAssignmentUtil {
+
+    public static final Set<String> OPERATOR_ADMIN_ROLES = new HashSet<>(
```

Comment:
We do not want to have this in code, but in:
- rdf4j-repository as triples in case of internal-auth (if needed)
- keycloak in case of keycloak-auth